StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkSettings.swift

```diff
@@ -21,10 +21,17 @@ import Foundation
         case ephemeral
     }
 
+    @_spi(STP) @frozen public enum LinkMode: String {
+        case linkPaymentMethod = "LINK_PAYMENT_METHOD"
+        case passthrough = "PASSTHROUGH"
+        case linkCardBrand = "LINK_CARD_BRAND"
+    }
```

Comment:
would be good to double-check this via backend link but looks good

(particularly important to be extra careful as this is payment SDK)

Reply:
also wroth double checking that this is indeed `@frozen`

for example, can backend add new values? and if they can what would happen client side?

Reply:
oh I guess the flatMap makes this very safe:

`(response["link_mode"] as? String).flatMap { LinkMode(rawValue: $0) }`

and since its `@_spi(STP)` it doesn't matter

I was worried about decoding errors

Reply:
Here's the enum definition: https://stripe.sourcegraphcloud.com/git.corp.stripe.com/stripe-internal/pay-server/-/blob/lib/consumer/payments_utils/data/link_mode.rb

Might not be a bad idea to get rid of `@frozen` since we don't know for sure things here won't change, but like you said, it doesn't make a huge difference because of `@spi_(STP)`